src/llm/llmnoderesources.hpp

```diff
@@ -47,6 +47,8 @@ struct LLMNodeResources {
 public:
     std::shared_ptr<ContinuousBatchingPipeline> cbPipe = nullptr;
     std::string modelsPath;
+    std::string device;
+    std::string pluginConfig;
```

Comment:
This type is not in line with CB lib. Shouldn't we have ov::AnyMap here and do some parsing of the string that comes from node options?

Reply:
Fixed.